Lib/test/test_importlib/test_main.py

```diff
@@ -0,0 +1,157 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+import textwrap
+import unittest
+import importlib
+import importlib.metadata
+
+from . import fixtures
+from importlib.metadata import _hooks
+
+
+class BasicTests(fixtures.DistInfoPkg, unittest.TestCase):
+    version_pattern = r'\d+\.\d+(\.\d)?'
+
+    def test_retrieves_version_of_self(self):
+        dist = importlib.metadata.Distribution.from_name('distinfo-pkg')
+        assert isinstance(dist.version, str)
```

Comment:
I'm cool with pytest style asserts, but I wonder if that fits in with the way the stdlib likes to do things.

Reply:
We should at least be internally consistent.  That might mean making changes to the standalone version.

Reply:
You raised this before. Here's [my response](https://gitlab.com/python-devs/importlib_metadata/merge_requests/34#note_127856165). Whatever we settle on, agreed it should go upstream first.